sct-commons/src/main/java/org/lfenergy/compas/sct/commons/scl/ln/LnKey.java

```diff
@@ -0,0 +1,38 @@
+// SPDX-FileCopyrightText: 2021 RTE FRANCE
+//
+// SPDX-License-Identifier: Apache-2.0
+package org.lfenergy.compas.sct.commons.scl.ln;
+
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import org.lfenergy.compas.scl2007b4.model.LN0;
+import org.lfenergy.compas.scl2007b4.model.TLN;
+
+import java.util.Optional;
+
+@Getter
+@EqualsAndHashCode
+public class LnKey {
+
+    private final String inst;
+    @EqualsAndHashCode.Exclude
+    private final String lnType;
+    private final String lnClass;
+    /** empty pour les LN0 */
+    private final Optional<String> prefix;
```

Comment:
Optional was mainly intended for return values from methods, and not for properties.

Reply:
I am absolutely not agree with this; but anyway I used String related with @massifben comment